sdk/core/core-http/test/mockHttp.ts

```diff
@@ -49,8 +49,7 @@ class FetchHttpMock implements HttpMockFacade {
 
   // returns the locally mocked fetch instance
   getFetch(): typeof node_fetch {
-    /// @ts-ignore
-    return this._fetch as typeof node_fetch;
+    return (this._fetch as unknown) as typeof node_fetch;
```

Comment:
Removing `@ts-ignore` exposed this error. I just used the editor's Code Action to auto-fix it but it seems to line up with https://microsoft.github.io/TypeScript-New-Handbook/everything/#type-assertions - maybe there's a cleaner way by updating `FetchMockSandbox`? I wasn't sure...

Reply:
I think this one might be from when @jeremymeng did #9880

I'm fine with your solution though, casting through unknown is closer to the truth than us ignoring the line altogether.

Reply:
Looks good!  I spent some time trying to make the mocked type compatible with type of node_fetch but it seemed non-trivial and at the end I chose to do cast instead for testing.